### PR TITLE
GH-43 SEP-0004 standard URI and keyword for referencing the default graph

### PR DESCRIPTION
**Pull request by jeenbroekstra**
This is a first draft (as yet incomplete).

### CODE REVIEW

**Review by afs — APPROVED** [review]
Looks good to merge.

**Review by JervenBolleman — APPROVED** [review]

**Comment by afs** [comment]
@jeenbroekstra 

It is in a state to merge so if you un-draft, it can be merged.

Discussion: Let's have that on the issue #43 so discussions on this and any later PRs are in the same place.

File: SEP/SEP-0004/sep-0004.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+| Blazegraph            | `<http://www.bigdata.com/rdf#nullGraph>` |
+| Eclipse RDF4J         | `<http://rdf4j.org/schema/rdf4j#nil>`    |
+| Stardog               | `<tag:stardog:api:context:default>`      |
+
+This clearly demonstrates a pragmatic need for a way to reference the default graph. Providing a standardized identifier for this purpose will make SPARQL queries more portable across different endpoint implementations.
+
+
+## Evidence of consensus
+
+See discussion on GH-43.
```
Comment (afs) [inline SEP/SEP-0004/sep-0004.md:39]: "GH-43" ==> "issue "#43".

File: SEP/SEP-0004/sep-0004.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+(TODO reach consensus on single best URI)
+
+- `<http://www.w3.org/1999/02/22-rdf-syntax-ns#defaultGraph>`.
+- `<about:default-graph>`
+
+and the proposed keyword is `DEFAULT`.
+
+## Motivation
+
+As raised in GH-43, the core problem is that the default graph is not referencable in dataset definitions.
```
Comment (afs) [inline SEP/SEP-0004/sep-0004.md:22]: Non-blocking but could we make this a link? e.g. "SPARQL-12 issue #43"

**Comment by jeenbroekstra** [comment]
Thanks all, done. It's pretty minimal but it gives us a starting point.

**Pull request merged**